pace.js

```diff
@@ -245,7 +245,9 @@
         this.el = document.createElement('div');
         this.el.className = "pace pace-active";
         document.body.className = document.body.className.replace(/pace-done/g, '');
-        document.body.className += ' pace-running';
+        if (!/pace-running/.test(document.body.className)) {
```

Comment:
Note: this was a fix already in the coffeescript but not yet compiled to the javascript.